### PR TITLE
db.csw.admin: fix readfp() removed from python 3.12

### PR DESCRIPTION
**Pull request by pesekon2**

### CODE REVIEW

**Comment by pesekon2** [comment]
Please review @krejcmat

File: src/gui/wxpython/wx.metadata/db.csw.admin/db.csw.admin.py [hunk 1]
```diff
@@ -283,7 +283,7 @@ def run(self, argv):
 
         if self.COMMAND not in ["post_xml"]:
             SCP = configparser.SafeConfigParser()
```
Comment (tmszi) [inline src/gui/wxpython/wx.metadata/db.csw.admin/db.csw.admin.py:285]:
```python
>>> SCP = configparser.SafeConfigParser()
<stdin>:1: DeprecationWarning: The SafeConfigParser class has been renamed to ConfigParser in Python 3.2. This alias will be removed in Python 3.12. Use ConfigParser directly instead.
```

**Comment by tmszi** [comment]
I think this addon is not maintained by original author long time. I made some changes ([since 2020](https://github.com/OSGeo/grass-addons/commits/master?after=8f43471d006c30d4bbac49973c28572d3cad8ec9+69&branch=master&path%5B%5D=grass7&path%5B%5D=gui&path%5B%5D=wxpython&path%5B%5D=wx.metadata&qualified_name=refs%2Fheads%2Fmaster)) which allow launch these addons with previous major version grass7 and current grass8, compatibilty with wxPython 4.0.7, 4.2.0, lazy module loading and [etc](https://github.com/OSGeo/grass-addons/commits/master/grass7/gui/wxpython/wx.metadata).

File: src/gui/wxpython/wx.metadata/db.csw.admin/db.csw.admin.py (hunk 1 above)
Comment (pesekon2) [reply src/gui/wxpython/wx.metadata/db.csw.admin/db.csw.admin.py:285]: I would recommend to change this in another commit. Having to separate ones makes it easier to find reasoning behind commits through `git blame`

Comment (tmszi) [reply src/gui/wxpython/wx.metadata/db.csw.admin/db.csw.admin.py:285]: Ok.

**Review by tmszi — APPROVED** [review]

File: src/gui/wxpython/wx.metadata/db.csw.admin/db.csw.admin.py (hunk 1 above)
Comment (pesekon2) [reply src/gui/wxpython/wx.metadata/db.csw.admin/db.csw.admin.py:285]: @tmszi: Funny thing - I apparently created a PR fixing the issue you had mentioned already more than a month ago as #919, but forgot about that. Going to merge that one too.

**Pull request merged**